website/static/js/profile.js

```diff
@@ -580,6 +582,15 @@ var ListViewModel = function(ContentModel, urls, modes) {
         return self.contents().length > 1;
     });
 
+    self.institutionObjectsEmpty = ko.pureComputed(function(){
+        for (var i=0; i<self.contents().length; i++) {
```

Comment:
This loop checks if there are any blank institution objects within the list to prevent users from continuing to add blank institutions to the view (madness!).  So if it finds any blank institutions, it exits the loop with a True, if not, its False City!